www/docs/client/links.md

```diff
@@ -25,9 +25,9 @@ const somePosts = await Promise.all([
 
 > The below examples assuming you use Next.js, but the same as below can be added if you use the vanilla tRPC client
 
-### Setting a maximum batch size
+### Setting a maximum URL length
 
-This limits the number of requests that can be sent together in batch ( useful to prevent the url from getting too large and run into [HTTP error 413](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/413) ).
+When sending batch requests, sometimes the URL can become too large causing [HTTP error 413](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/413). The `maxURLLength` option will limit the number of requests that can be sent together in a batch. This value defaults to 2,083 but you may wish to increase or decrease this dependent on your deployment environment & use case. You can read more about URL lengths [here](https://www.sistrix.com/ask-sistrix/technical-seo/site-structure/url-length-how-long-can-a-url-be).
```

Comment:
I would've thought it would be [HTTP error 414](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/414) "URI Too Long." Maybe we should also make a note for that?

Reply:
Hmm, it also looks like it will sometimes result in a 404 error.. https://github.com/trpc/trpc/discussions/1792

Reply:
Maybe something like this:
```
When sending batch requests, sometimes the URL can become too large causing HTTP errors like ["413 Payload Too Large"](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/413), ["414 URI Too Long"](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/414), and ["404 Not Found"](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404).
```

Just to make it easier to find the section if someone gets the error.